src/cmd/_main.cpp

```diff
@@ -182,6 +217,10 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	if (clean_result && mode != LIST_DIR && mode != SHOW_FILE) clean_result = false;
+
+	if (!clean_result) print_copyright();
```

Comment:
Copyright should not be optional, in my opinion. Up to @mwl4.

Reply:
This is that external parsers get a guaranteed clean result, without trying to separate the copyright from the output data.